index.js

```diff
@@ -154,135 +259,13 @@ console.log(count)
- let p2 = document.querySelector('#name_player2')
- p2.textContent = player2.name
-
- //place number of remaining ships in the id ships_player2
-  let shipsP2 = document.getElementById('ships_player2')
-  shipsP2.textContent = player2.shipCount;
-
- //set up the ships
- function boardPlacement () {
-   let ships=0;
-   let xPos;
-   let yPos;
-
-   while(ships < player1.shipCount){
-     xPos = Math.floor(Math.random() * 4);
-     yPos = Math.floor(Math.random() * 4);
-     if(parseInt(player1.gameBoard[xPos][yPos])!==1){
-       player1.gameBoard[xPos][yPos] = 1;
-       ships++;
-     }
-   }
-   ships=0;
-   while(ships < player2.shipCount){
-     xPos = Math.floor(Math.random() * 4);
-     yPos = Math.floor(Math.random() * 4);
-     if(parseInt(player2.gameBoard[xPos][yPos])!==1){
-       player2.gameBoard[xPos][yPos] = 1;
-       ships++;
-     }
-
-   }
-   console.log(player1.gameBoard,player2.gameBoard) ;
- }boardPlacement();
-
-
-//let count=0
-
-// function playerSwitch (){//MJF:when uncommented, this function is coming up before the board is formed.
-//   while(board_Player2){
-//     //currentPlayer.textContent = turnPlayer
-//     if(+(count%2)===0){
-//       currentPlayer = player1.name;
-//       opponentPlayer = player2.name;
-//
-//       alert(`Your turn ${player2.name}`);
-//       //break;
-//     } else{
-//       opponentPlayer = player1.name;
-//       alert(`Your turn ${player1.name}`);
-//     }
-//
-//   }
-//
-// }
-
-
-// let turnPlayer;
-// function turn (){
-//   turnPlayer = document.getElementById('turn_player');
-//   turnPlayer.textContent
-//
-//   if (turnPlayer === currentPlayer){
-//     turnPlayer.textContent;//MJF: thought the value would display but it's not.
-//     console.log('yo')
-//   }else if(turnPlayer === opponentPlayer){
-//     turnPlayer.textContent;
-//   }
-//
-//   console.log(turnPlayer)
-// }turn()
-
-
-//have a function to declare the winner(look at the other version)
-let winner;
-function win(){//MJF: not sure why this would not work
-console.log('Win function running!')
-
-if(parseInt(shipsP1.textContent)==0){//maybe not working bc it thinks it is a string so will never happen
-  winner = currentPlayer
-  //alert(`${player2.name} is the winner!`);
-}else if(parseInt(shipsP2.textContent)==0){
-  //alert(`${player1.name} is the winner!`);
-  winner = currentPlayer
-}//else{
-//   break;
-// }
-alert (`The winner is ${winner}`)
-console.log(typeof shipsP1.textContent)
-}//win()
-
 
 //reset and new game buttons
-function reSet(){//try to get the reset button to connect to the Dom by connecting it to the function with the boardPlacement
+function reSet(){
   let resetBtn = document.createElement('button')
   let button = document.getElementById('buttons').appendChild(resetBtn)
   resetBtn.textContent = "Reset"
-  board_Player1.style.display ='block'
-  board_Player2.style.display ='block'
+
   resetBtn.addEventListener("click", () => {
```

Comment:
this button was working earlier today but now its not reseting the board. the lives do reset fine. I am not sure why its not working. it's 3am now and i can't look at it anymore. if someone is reviewing this  at some point, please let me know where the problem is. thanks